### ISSUE TITLE
pull access denied for mosaik/orch-base

### ISSUE THREAD

**Issue by stlch** [ERIGrid2/mosaik-docker-demo#1]
Hi,

I'm quite new to especially Docker, so I might just miss configured something, but when running the monolithic demo the command "build_sim_setup ." runs into the following error:

`:~/mosaik-docker-demo/monolithic$ build_sim_setup .
Sending build context to Docker daemon    424kB
Step 1/11 : FROM mosaik/orch-base:v1
building simulation setup failed:
pull access denied for mosaik/orch-base, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
run "check_sim_setup" for details

:~/mosaik-docker-demo/monolithic$ check_sim_setup
simulation setup is valid: /mosaik-docker-demo/monolithic`

I also did not manage to find mosaik/orch-base via docker search.

Is the repository still up, do I need the specify some login information somewhere, or what else can I do to get the demo running?

Thanks in advance,
Malte

**Comment by ewidl** [comment]
You are right, this image is not (yet) published. But you can easily build it yourself using the Dockerfile and script available [here](https://github.com/ERIGrid2/mosaik-docker/tree/master/dockerfiles).

**Comment by stlch** [comment]
Thanks for the quick reply, that worked for me.

Just One Thing:

In line 7 in [Dockerfile_base](https://github.com/ERIGrid2/mosaik-docker/blob/master/dockerfiles/Dockerfile_base) the mosaik-api version needs to be changed to 2.4.2, since pip runs into an error when trying version 2.4.1:

pip install mosaik-api==2.41
ERROR: Could not find a version that satisfies the requirement mosaik-api==2.4.1 (from versions: 2.0a1, 2.0a2, 2.0a3, 2.0a4, 2.0, 2.1, 2.2, 2.2.1, 2.3, 2.4.1, 2.4.2, 3.0.0)
ERROR: No matching distribution found for mosaik-api==2.4.1
